src/logic/map_objects/tribes/building.cc

```diff
@@ -387,7 +388,7 @@ Flag& Building::base_flag() {
 uint32_t Building::get_playercaps() const {
 	uint32_t caps = 0;
 	const BuildingDescr& tmp_descr = descr();
-	if (tmp_descr.is_destructible()) {
+	if (tmp_descr.is_destructible() && !is_indestructible()) {
```

Comment:
How about renaming `is_indestructible()` to `is_destruction_blocked()`? This would make it clearer that the new temporary property is not the inverse of the permanent destructible property.